articles/architecture-scenarios/b2c/user-logout.md

```diff
@@ -0,0 +1,68 @@
+---
+title: User Logout
+description: What to consider for user logout
+toc: true
+topics:
+    - b2c
+    - ciam
+    - logout
+    - sessions
+contentType: concept
+useCase:
+    - user-logout
+---
+# User Logout
+
+[Logout](/logout) is the act of terminating an authenticated session when they're no longer needed, minimizing the likelihood that unauthorized parties can "take over" the session. This is achieved by provisioning a logout option on the user interface you provide to your users.
+
+Multiple types of sessions can be created when a user logs in (e.g., local application sessions, Auth0 session, third-party Identity Provider sessions), and you will need to determine which of these sessions need to be terminated when the user clicks **Logout**.
+
+## Design considerations
+
+When configuring logout behavior, you'll need to consider:
+
+* Which sessions should be terminated when the user initiates logout
+* What information to provide to users as confirmation of the sessions terminated
+* Where the users should be redirected to after logout completes
+* Whether actions take by the user elsewhere (such as in another application) need to be accounted for to [provide a single logout experience](/logout/guides/logout-applications#single-sign-out-configuration-example)
+* How long you want sessions to last in the event that users do not trigger the logout process
+
+### Types of logout
+
+Given the varying types of sessions that can be created whenever a user logs in, there are several types of logout possible:
+
+* Local application logout to end the application session
+* Auth0 logout to terminate the Auth0 session
+* Federated logout to terminate the Auth0 session and propagate the logout request to a remote, third-party Identity Provider
+* Global, or single, logout to end the Auth0 session and send a logout request/notice to applications relying on the Auth0 session
+
+Please note that the examples above assume the use of [OIDC](/protocols/oidc).
```

Comment:
I don't think there's anything OIDC specific here, so perhaps just remove this altogether?
```suggestion

```